fedimint-server/src/consensus/server.rs

```diff
@@ -587,6 +587,7 @@ impl ConsensusServer {
             .expect("This is the only place where we write to this key");
     }
 
+    #[instrument(target = "fm::consensus", skip(self, item), level = "info")]
```

Comment:
I wish I could do constants we use, but it needs https://github.com/tokio-rs/tracing/pull/2941